src/pid/CustomizeControlMode.java

```diff
@@ -0,0 +1,7 @@
+package pid;
+
+public enum CustomizeControlMode {
```

Comment:
Customize is a verb, enums should be a noun, Just `ControlMode` would suffice, but since it's taken I'd go for `PIDControlMode`.